doc/testing.md

```diff
@@ -43,6 +43,29 @@ containing the core JMH and transitive dependencies. The recommended
 dependencies can be retrieved by running `sh make/devkit/createJMHBundle.sh`,
 after which `--with-jmh=build/jmh/jars` should work.
 
+When tests fail or timeout, jtreg runs its failure handler to capture necessary
+data from the system where the test was run. This data can then be used to
+analyze the test failures. Collecting this data involves running various commands
+(which are listed in files residing in `test/failure_handler/src/share/conf`)
+and some of these commands use `sudo`. If the system's `sudoers` file isn't
+configured to allow running these commands, then it can result in password being
+prompted during the failure handler execution. Typically, when running locally,
+collecting this additional data isn't always necessary. To disable running the
+failure handler, use `--enable-jtreg-failure-handler=no` when running `configure`.
+If, however, you want to let the failure handler to run and don't want to be prompted
```

Comment:
Most of this file is kept close to 80 chars per line, at least in the text blocks. Up until here, you have kept with this style, but the additions below get increasingly wider, which looks a bit weird. Would you mind tiding it up a bit?

Reply:
Hello Erik, I've now updated the file to fix the line lengths.